app/views/shops/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>All Bubble Tea Shops</h1>
+
+<ul>
+    <% @shops.each do |shop| %>
+        <li>
+            <%= shop.name %>
+        </li>
+    <% end %>
+</ul>
```

Comment:
most of the new files in this PR look to have a tab size of 4, whereas the rest of the files have a tab size of 2. This should be consistent across members of a project - I'd recommend everyone updating their VSCode settings to use tab size 2!